mobile/mysterium/proposals_manager.go

```diff
@@ -208,9 +210,15 @@ func (m *proposalsManager) getFromRepository(req *GetProposalsRequest) ([]propos
 
 	// Ideally api should allow to pass multiple service types to skip noop
 	// proposals, but for now just filter in memory.
+	serviceTypesDict := map[string]bool{
```

Comment:
Just `serviceTypes` without `Dict` will be the same clear but shorter. We already know that this is a map, no need to put it into the variable name.